docs/_content/standaard/standaard.md

```diff
@@ -100,36 +100,35 @@ Authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiIsImNsaWVudF9pZGVudGlmaW
 
 Client en server maken gebruik van `shared secret` om het JWT te signen, met
 het HMAC SHA-256 algoritme. Iedere client MAG een eigen secret hebben. De
-server MOET aan de hand van de `client_identifier` key in de JWT header
+server MOET aan de hand van de `client_id` key in de JWT payload
 de bijhorende secret opvragen. De server MOET met het juiste shared secret
 het JWT valideren tegen tampering.
 
-De ZGW claims in de JWT-payload worden genamespaced onder `zds`.
-
-De claims bevatten de scopes als lijst van strings, waarbij de `scopes` key
-gebruikt wordt.
-
-De `zaaktypes` claim MOET gebruikt worden om zaakgegevens te limiteren. Indien
-deze claim ontbreekt, `null` is of een lege lijst, dan is het VERBODEN om
-zaakgegevens te ontsluiten. Een speciale waarde van `['*']` drukt uit dat alle
-zaaktypes toegestaan zijn.
-
 Voorbeeld van een payload:
 
 ```json
 {
-    "zds": {
-        "scopes": [
-            "zds.scopes.zaken.aanmaken"
-        ],
-        "zaaktypes": [
-            "https://haarlem.ztc.nl/api/v1/zaaktypen/123",
-            "https://haarlem.ztc.nl/api/v1/zaaktypen/124",
-        ]
-    }
+    "iss": "voorbeeld-consumers",
+    "iat": 1556898201,
+    "client_id": "dsh5thqAzL6I9SC5IPgR"
```

Comment:
Merk dat `client_id` en `applicatieId` een beetje hetzelfde zijn. Wellicht 1 lijn in trekken of dit benoemen ergens?

Reply:
we hebben tot nu toe overal `client_id` gebruikt. Waar komt `applicatieId` plots ter sprake?

Reply:
Ik zie het al, in audit trail stukje:

> `applicatieId` (verplicht)
>
> Unieke identificatie van de applicatie, binnen de organisatie. Deze komt 
> overeen met het `client_id` attribuut uit de JWT.

Semantisch is er 1 applicatie, maar die kan zich met verschillende client IDs identifieren (1 client ID per backend, om hergebruik van credentials te vermijden). M.i. moet dus het stukje in audit trail aangepast worden.

Reply:
check, hoewel niet heel Nederlands.